.laminas-ci.json

```diff
@@ -8,7 +8,12 @@
         "sqlite3",
         "sqlsrv"
     ],
-    "ignore_php_platform_requirements": {
-        "8.1": true
-    }
+    "exclude": [
+        {
+            "name": "PHPUnit on PHP 8.1 with latest dependencies"
+        },
+        {
+            "name": "PHPUnit on PHP 8.1 with lowest dependencies"
```

Comment:
This really shouldn't be excluded :|

Reply:
This PR would fail otherwise, PHP 8.1 is not supported yet.

The `"ignore_php_platform_requirements": {"8.1": true}` syntax is not working. 

PHP 8.1 Tests are still generated https://github.com/laminas/laminas-db/actions/runs/1887918057

Reply:
We do though :thinking: 

https://github.com/laminas/laminas-db/blob/feeb1ffe83793d33305950a48c0eb9e540976ef6/composer.json#L28

Reply:
I meant PHP 8.1 CI matrix: Unable to locate package `php8.1-sqlsrv`

Because it can't find the packing, it automatically fails early, before reaching the steps where it uses the `services` defined for the container.

Reply:
I thought about that, but in `laminas-ci.json` the `extensions` we define for **all** PHP version, cause the script to fail on PHP 8.1 because one or more of the extensions doesn’t exist.

``` 
"extensions": [
        "pdo-mysql",
        "pdo-pgsql",
        "pdo-sqlite",
        "mysqli",
        "pgsql",
        "sqlite3",
        "sqlsrv"
    ],
```

if we remove this then we will lose those extensions on all PHP versions.

Reply:
Actually, we need to update the laminas-continuous-integration-action. MS released v5.10.0 of the sqlsrv extension, and it has PHP 8.1 support. I'll add that to the CI action, and then we should be able to test 8.1 here.